Sources/QuickTests/Helpers/QCKSpecRunner.m

```diff
@@ -12,11 +12,6 @@
         if ([suite respondsToSelector:@selector(runTest)]) {
             [suite runTest];
             result = suite.testRun;
-        } else {
-#pragma clang diagnostic push
-#pragma clang diagnostic ignored "-Wdeprecated-declarations"
-            result = [suite run];
-#pragma clang diagnostic pop
```

Comment:
Yup, looks like Apple XCTest removed this API; see https://github.com/apple/swift/pull/2645. But don't we need to call the newer API, `-[XCTest runTest]` here? I think we can then grab the result off of the test case object.